### PR TITLE
updated models

### PR DESCRIPTION
**Pull request by fathimasulthanaas**

### CODE REVIEW

File: server/MyDoctor/MyDoctor.Domain/Models/DoctorMaster.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace MyDoctor.Domain.Models
+{
+    public class DoctorMaster
+    {
+        public int Id { get; set; }
+        public string UserId { get; set; }
+        public ApplicationUser ApplicationUser{ get; set; }
+        public string DepartmentId { get; set; }
+        public Department Department { get; set; }
```
Comment (mishelshaji) [inline server/MyDoctor/MyDoctor.Domain/Models/DoctorMaster.cs:16]: Format the code.

File: server/MyDoctor/MyDoctor.Domain/Models/DoctorMaster.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        [StringLength(20)]
+        public string DoctorLicenseNo { get; set; }
+        public DateTime? Dob { get; set; }
```
Comment (mishelshaji) [inline server/MyDoctor/MyDoctor.Domain/Models/DoctorMaster.cs:20]: Add a blank line above.

File: server/MyDoctor/MyDoctor.Domain/Models/DoctorMaster.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+        [StringLength(10)]
+        public string? Gender { get; set; }
+        public int Status { get; set; }
```
Comment (mishelshaji) [inline server/MyDoctor/MyDoctor.Domain/Models/DoctorMaster.cs:24]: Add a blank line above. Also, why not use enum instead?

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: server/MyDoctor/MyDoctor.Domain/Models/Appointment.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    public class Appointment
+    {
+        public int Id { get; set; }
+        public string PatientMasterId { get; set; }
+        public PatientsMaster PatientsMaster { get; set; }
+        public string DoctorId { get; set; }
+        public DoctorMaster DoctorMaster { get; set; }
+        public DateTime DateTime { get; set; }
+        public DateTime CreatedDateTime { get; set; }
+        public int Status { get; set; }
```
Comment (mishelshaji) [inline server/MyDoctor/MyDoctor.Domain/Models/Appointment.cs:18]: Why not change to an enum?

File: server/MyDoctor/MyDoctor.Domain/Models/Consultation.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace MyDoctor.Domain.Models
+{
+    public class Consultation
+    {
+        public int Id { get; set; }
+        public string AppointmentId { get; set; }
+        public Appointment Appointment { get; set; }
+        public string DiseaseId { get; set; }
+        public Disease Disease { get; set; }
```
Comment (mishelshaji) [inline server/MyDoctor/MyDoctor.Domain/Models/Consultation.cs:16]: Format the code.

File: server/MyDoctor/MyDoctor.Domain/Models/DoctorMaster.cs [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace MyDoctor.Domain.Models
+{
+    public class DoctorMaster
+    {
+        public int Id { get; set; }
+        public string UserId { get; set; }
+        public ApplicationUser ApplicationUser { get; set; }
+        public string DepartmentId { get; set; }
+        public Department Department { get; set; }
```
Comment (mishelshaji) [inline server/MyDoctor/MyDoctor.Domain/Models/DoctorMaster.cs:16]: Format the code.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: server/MyDoctor/MyDoctor.Domain/Models/Appointment.cs (hunk 4 above)
Comment (fathimasulthanaas) [reply server/MyDoctor/MyDoctor.Domain/Models/Appointment.cs:18]: enum not needed

File: server/MyDoctor/MyDoctor.Domain/Models/Consultation.cs [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+        public string AppointmentId { get; set; }
+
+        public Appointment Appointment { get; set; }
+
+        public string DiseaseId { get; set; }
+
+        public Disease Disease { get; set; }
+
+
```
Comment (mishelshaji) [inline server/MyDoctor/MyDoctor.Domain/Models/Consultation.cs:22]: Remove the blank line.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**